mingw-w64-minizip-ng/PKGBUILD

```
@@ -0,0 +1,82 @@
+# Maintainer: Andrew Sun <adsun701@gmail.com>
+
+_realname=minizip-ng
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}2")
```

```suggestion
pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```

Or was this supposed to be `minizip-ng2`?

---

That 2 looks like a mistake. will remove.

---

Fixed.